### PR TITLE
porting callable functions

### PR DESCRIPTION
**Pull request by thatfiredev**
TODO:
- [x] Fill "Getting Started" section on README.md

### CODE REVIEW

File: 2nd-gen/callable-functions/functions/index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {onCall, HttpsError} = require("firebase-functions/v2/https");
+const {getDatabase} = require("firebase-admin/database");
```
Comment (jhuleatt) [inline 2nd-gen/callable-functions/functions/index.js:18]: great use of the modular admin SDK. I'm going to add a note in the devrel plan to make sure everyone does this.

Comment (jhuleatt) [reply 2nd-gen/callable-functions/functions/index.js:18]: added

File: 2nd-gen/callable-functions/functions/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+TODO
+
+
+Support
+-------
+
+- [Firebase Support](https://firebase.google.com/support/)
+
+
```
Comment (jhuleatt) [inline 2nd-gen/callable-functions/functions/README.md:24]: We can keep this very minimal for now. Maybe something like:

```suggestion
[Read more about Cloud Functions for Firebase](https://firebase.google.com/docs/functions/)


Getting Started
---------------

1. Install dependencies with `npm install` and deploy with `firebase deploy --only functions`
1. Set up a client (import the functions client SDK, initializeApp, init the functions sdk)
2. Call the function


```

**Review by egilmorez — APPROVED** [review]
Thanks!  These are the include tags we know and love from the v1 version.

**Comment by egilmorez** [comment]
Actually Rosario, I just realized: we want to prepend all the the tag names with "v2" for easier identification in the snippet registry.  Can you update the tags in index.js?  Thanks!

**Comment by egilmorez** [comment]
Actually Rosario, I just realized: we want to prepend all the the tag names with "v2" for easier identification in the snippet registry.  Can you update the tags in index.js?  Thanks!

**Comment by thatfiredev** [comment]
Done @egilmorez

**Review by egilmorez — APPROVED** [review]
LG, thanks!

**Pull request merged**